### ISSUE TITLE
Add i18n tags to all Django templates

### ISSUE THREAD

**Issue by brylie** [CiviWiki/OpenCiviWiki#876]
As we port templates from the SPA app over to regular Django templates, ensure that we add localization support for all static texts. Prior to the 2.0 release, review all templates to make sure the internationalization is complete.

**Comment by mharyam** [comment]
Hello @brylie, 
Can I take this?

**Comment by brylie** [comment]
@mharyam yes please :-) I'll assign you to the issue. Please let us know how we can be of assistance.

**Comment by mharyam** [comment]
Thank you, I will.

**Comment by mharyam** [comment]
Hello @brylie,

For this task, do you think it is ideal I add template tag {%trans%} to all the individual templates?

**Comment by brylie** [comment]
@mharyam, yes. Follow the [guidelines from Django documentation](https://docs.djangoproject.com/en/3.1/topics/i18n/).

**Comment by mharyam** [comment]
Hello @brylie 
I have created my first pull request for this task
Link: https://github.com/CiviWiki/OpenCiviWiki/pull/976

I added translation to the landing page, header and footer 
If it is fine, I will proceed to the next one 

Thank you.

**Comment by brylie** [comment]
Thank you so much @mharyam! :smiley: 

I've reviewed your pull request and made some minor suggestions with code changes that you can automatically accept.

**Comment by mharyam** [comment]
Hello @brylie 

Thanks a lot for taking out time to review! I have accepted all the suggestions. 

It looks like you still need to merge the pull request, so it can be updated to the main branch

Thank you.

**Comment by mharyam** [comment]
Thank you @brylie  for taking out time to approve the pull request. 

Do you think there are other pages I need to add translation to?

**Comment by brylie** [comment]
Basically, just look for a `templates` subfolder in any of the Django app folders. For example, there are several [accounts-related templates](https://github.com/CiviWiki/OpenCiviWiki/tree/develop/project/accounts/templates/accounts) that need i18n tags.

**Comment by rsk2** [comment]
Hi @brylie, I'd like to help with this issue. Please let me know if there is any work that still needs to be done for this issue.
